site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/rsf/GroupListProducer.java

```diff
@@ -126,17 +133,29 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams,
 				String groupId = group.getId();
 				UIBranchContainer grouprow = UIBranchContainer.make(deleteForm, "group-row:", group.getId());
 				
-				UIOutput.make(grouprow, "group-title-label", group.getTitle());
+				String groupTitle = Validator.escapeHtml(group.getTitle());
```

Comment:
I have made a pull request (https://github.com/sakaiproject/sakai/pull/4093) to resolve this. Thanks @stendler